main.py

```diff
@@ -131,7 +132,8 @@ def get_daily_cash_and_chosen_index(transactions: pd.DataFrame, historical_data:
 
     for _, row in transactions.iterrows():
         if not row.Symbol.strip():
-            if row.Action != "Interest" and "Funds Received" in row.Description:
```

Comment:
> `"Funds Received"`
Maybe you also want to add the counterpart of when funds are withdrawn.

Reply:
Feel free to add this feature.

Reply:
I don't know what the word would be.